Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/Mappings.kt

```diff
@@ -75,6 +75,9 @@ fun PrimitiveType.getKotlinType(declarationMapper: DeclarationMapper): KotlinCla
         else -> TODO(this.toString())
     }
 
+    is VectorType -> {
+        KotlinTypes.nativeVector
```

Comment:
It makes sense to add an assertion checking that this is 16-byte vector.

Reply:
And better mention that it's 128 bits in the name.